src/Components/Aspire.Microsoft.Azure.Cosmos/AspireAzureCosmosDBExtensions.cs

```diff
@@ -108,6 +109,19 @@ public static class AspireAzureCosmosDBExtensions
         if (serviceKey is null)
         {
             builder.Services.AddSingleton(_ => ConfigureDb());
+
```

Comment:
In line 104, please remove `clientOptions.LimitToEndpoint = true;` this should never be set in any general content, even in the case of the emulator, users might copy this and think this is ok.

I know this came from an official doc (I found the source) and I also corrected it there.

Reply:
If I remove this, it seems to stop the code working. It just hangs.

Reply:
It is not hanging, it probably is facing an HttpRequestException and retrying. This flag turns off the retries. HttpRequestException is normally related to some connectivity issue with the endpoint being passed. Was it localhost?